azure/package.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
why is this file added. Seems unrelated. Bad merge?

Reply:
Yes, bad merge. Intending to delete this before merging into main.